### DISCUSSION TITLE
How to Convert Color from SolidColorBrush in XAML?

### DISCUSSION THREAD

**Discussion by xinghaiyege** [AvaloniaUI/Avalonia#13583] [Q&A]
How to Convert `Color` from `SolidColorBrush` in `XAML`?

I want to set the `Color` of the `Effect` property through the `Background` property in the `Style`.

The value type of the `Background` property is `SolidColorBrush`.

**Comment by rabbitism** [comment]
You should use ValueConverter

**Comment by Coloryr** [comment]
use #xxxxx ?
